Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -939,3 +940,31 @@
+  id: UplinkHardsuitBloodRedModsuit
+  name: uplink-modsuit-bloodred-name
+  description: uplink-modsuit-bloodred-desc
+  icon: { sprite: /Textures/_Goobstation/Clothing/OuterClothing/Modsuits/bloodredmodsuit.rsi, state: chestplate }
+  productEntity: BloodredModsuitBackpack
+  discountCategory: rareDiscounts
+  discountDownTo:
+    Telecrystal: 15
+  cost:
+    Telecrystal: 25
+  categories:
+  - UplinkWearables
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)